hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -293,8 +293,13 @@ def _get_item_info(self, path):
             (category_type, _) = mimetypes.guess_type(filename)
 
             if category_type:
-                # category members are used in constructing items which contain utf-8 encoded strings, hence the encode() below
-                category = category_type.split("/")[0].encode("utf-8")
+                # make sure category_type is utf-8 encoded
```

Comment:
Suggest some more description here for background on why we do this. i.e why are we ensuring this is a utf-8 encoded Unicode string?